plugins/module_utils/arguments.py

```diff
@@ -38,7 +38,6 @@
             timeout=dict(
                 type="float",
                 required=False,
-                no_log=True,
```

Comment:
For some reason this was effecting vm_info module?
I don't know how it's possible, but certain values were returned as *****.

Reply:
If timeout had value "bla", than every "bla" in ansible output would be replaced by ****. Maybe timeout had value "0" or "None", those are common, so a lot of unrelated text would be replaced.

Reply:
Nice you removed that line.